aurci/general.py

```diff
@@ -49,9 +49,19 @@ def build_metainfo_dict():
             pkg_list = d.get('release', {'packages': [repo]}).get('packages', [repo])
             for pkg in pkg_list:
                 siblings = len(pkg_list)-1
+                # TODO: Replace hard-coded melodic
                 pkgname = 'ros-melodic-{}'.format(re.sub('_', '-', pkg))
+                pkg_url = '/'.join([re.sub(r'\.git', '', src),
+                                    'archive/release/melodic',
```

Comment:
You can only  guess the url paths on ros-gbp, so that's a bad idea. IMHO a specific instead of a general url generation always will be.

Reply:
I don't think, so, as those will be generated automatically, by the bloom release process, right?

Reply:
`archive/release/melodic`  is specific to the release pages of ros-gbp. That's the point I was trying to make.